com.mojang.Minecraft.json

```diff
@@ -0,0 +1,65 @@
+{
+  /* GTK+ is needed by JavaFX. */
+  "app-id": "com.mojang.Minecraft",
+  "runtime": "org.freedesktop.Sdk.Extension.openjdk9",
```

Comment:
This should be in `"sdk-extensions"` not set as the runtime.

Reply:
And it needs a module which copies the JDK: https://github.com/flathub/org.octave.Octave/blob/master/org.octave.Octave.json#L320